repo/packages/J/jupyter/0/config.json

```diff
@@ -0,0 +1,480 @@
+{
+  "type": "object",
+  "properties": {
+    "service": {
+      "type": "object",
+      "description": "The JupyterLab Notebook Service for Mesosphere DC/OS comes with a variety of pre-installed packages. Further information can be found here https://github.com/dcos/examples/blob/dcos-labs-jupyter-service/jupyter/1.11/README.md",
```

Comment:
We sometimes call it jupyter service and here JupyterLab.

Reply:
this is taken from https://github.com/dcos-labs/dcos-jupyter-service/releases as we call it here currently `JupyterLab Notebook Service` - the repo itself is called `dcos-jupyter-service` and the docker hub images `dcos-jupyter` with the universe packag also called `jupyter` - which is also a tag when searching it within `Catalog`.  @vishnu2kmohan would it make sense to you to change the release notes into `Jupyter Notebook Service` and @joerg84  would you agree with this naming convention?

Reply:
sgtm